SecondViewController/SecondViewController/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AppDelegate.swift
+//  SecondViewController
+//
+//  Created by Admin on 03.07.2021.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
файнал, и удали отсюда все, что не используешь
```suggestion
final class AppDelegate: UIResponder, UIApplicationDelegate {
```

Reply:
Удалил этот проект. Случайно его сделал.